lib/active_merchant/billing/gateways/paystation.rb

```diff
@@ -97,9 +100,9 @@ def add_customer_data(post, options)
         def add_invoice(post, options)
           requires!(options, :order_id)
 
-          post[:ms] = options[:order_id]     # "Merchant Session", must be unique per request
+          post[:ms] = options[:order_id] +"-"+ Time.now.to_f.to_s    # "Merchant Session", must be unique per request
```

Comment:
Please don't append time like this; it's expected that `order_id` must be unique.